.travis.yml

```diff
@@ -3,10 +3,6 @@ sudo: false
 cache: bundler
 bundler_args: "--without development"
 
-before_install:
```

Comment:
can you explain why you are doing this in the commit message?

Reply:
Sure. I opened this PR quite a while ago, but I think the issue was, that the build was failing: https://travis-ci.org/github/stevenosloan/slack-notifier/builds/494350577

When I removed these lines, the build was green again: https://travis-ci.org/github/stevenosloan/slack-notifier/builds/494352716

I checked the Travis docs an apparently these lines are required if Bundler 2.x should be used in older Ruby versions (https://docs.travis-ci.com/user/languages/ruby/#bundler-20). Not sure if that is really required though.

Reply:
would be good to rebase and verify that the build still fails without this - maybe worth pulling into a separate PR if it does?

Reply:
Dropped the commit and the build still fails: https://travis-ci.com/github/fabsrc/slack-notifier/builds/181107631

But if it can be merged with the failing build, it's fine for me.

Reply:
I'm new to getting involved with this repo - gonna try and chat to the current maintainer and see how we can get this merged (probably with the travisCI fix one way or another). thanks for your patience - I'll get back to you when I can